modules/core/src/main/scala/org/tesselation/domain/cluster/storage/ClusterStorage.scala

```diff
@@ -1,10 +1,14 @@
 package org.tesselation.domain.cluster.storage
 
+import org.tesselation.schema.cluster
+import org.tesselation.schema.cluster.TrustInfo
 import org.tesselation.schema.peer.{Peer, PeerId}
 
 import com.comcast.ip4s.{Host, Port}
 
 trait ClusterStorage[F[_]] {
+  def updateTrust(trustUpdates: cluster.InternalTrustUpdateBatch): F[Unit]
+  def getTrust(): F[Map[PeerId, TrustInfo]]
```

Comment:
if that's disjoint to Peer, imo you should create a `trust` or `reputation` package with the `TrustStorage` inside. We'd like to avoid mixing things when not necessary

Reply:
Done